app/src/main/res/layout/question_player_content_item.xml

```diff
@@ -0,0 +1,32 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="htmlContent"
+      type="CharSequence" />
+  </data>
+
+  <FrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="12dp"
+    android:layout_marginTop="28dp"
```

Comment:
The top margin in mocks is 38dp which can be (36 or 40). Now your recyclerview contains a padding of `16dp` and therefore this top margin should be either `20dp` or `24dp`

Reply:
removed recycler view padding as per other interactions spacing updated margin.. hence  
 ```
android:layout_marginTop="36dp"
android:layout_marginBottom="28dp"
```
bottom margin will be added with margin 8dp of upcoming items in recyclerview